main.py

```
@@ -1,44 +1,43 @@
+            if max(value_yes, value_no) > 0 and count < 5:
+                side = 'YES' if value_yes > value_no else 'NO'
+                vol = m.get('volume24hrs', 0)
+                msg += f"â€¢ {m['question'][:50]}...\n  {side}: {max(value_yes,value_no):.1%} Vol ${vol:,.0f}\n\n"
+                count += 1
+        bot.reply_to(message, msg or "No high value bets now.")
+    except Exception as e:
+        bot.reply_to(message, f"Error: {e}")
+
+@bot.message_handler(commands=['oscars'])
+def oscars(message):
+    bot.reply_to(message, "ðŸ“½ï¸ OSCARS Sinners: Yes 18% (72% value) RECOMMEND! Vol $23M [page:3]\np olymarket.com/event/oscars-2026-best-picture-winner")
```

Ruff (0.15.4)

[warning] String contains ambiguous `¸` (CEDILLA). Did you mean `,` (COMMA)?

(RUF001)